uuid breaking changes

go get -u github.com/goadesign/goa/... returns the following errors:
```
go get -u github.com/goadesign/goa/...
# github.com/goadesign/goa/uuid
../../../../../github.com/goadesign/goa/uuid/uuid.go:18:24: multiple-value uuid.NewV4() in single-value context
# github.com/goadesign/goa/design
../../../../../github.com/goadesign/goa/design/random.go:63:2: too many arguments to return
        have (uuid.UUID, error)
        want (uuid.UUID)
```

Changes in the uuid dependency package github.com/satori/go.uuid cause the problem. uuid.NewV4() now returns multiple values:
https://github.com/satori/go.uuid/commit/0ef6afb2f6cdd6cdaeee3885a95099c63f18fc8c

goa/design/random.go and goadesign/goa/uuid/uuid.go will need to be updated to handle the added return value.

Per the commit comment, the old (single value return) can be emulated by wrapping the calls to uuid.NewV4() in a uuid.Must() call:
`uuid.Must(uuid.NewV4())`

--- a/design/random.go
+++ b/design/random.go
@@ -60,7 +60,7 @@ func (r *RandomGenerator) DateTime() time.Time {
 
 // UUID produces a random UUID.
 func (r *RandomGenerator) UUID() uuid.UUID {
-	return uuid.NewV4()
+	return uuid.Must(uuid.NewV4())
 }
 
 // Bool produces a random boolean.

--- a/uuid/uuid.go
+++ b/uuid/uuid.go
@@ -15,7 +15,7 @@ func FromString(input string) (UUID, error) {
 
 // NewV4 Wrapper over the real NewV4 method
 func NewV4() UUID {
-	return UUID(uuid.NewV4())
+	return UUID(uuid.Must(uuid.NewV4()))
 }
 
 // String Wrapper over the real String method

